saiph/inverse_transform_test.py

```diff
@@ -31,6 +31,13 @@ def test_get_random_weighted_columns(weights: List[float], expected_index: int)
     assert result.values[0] == expected_index
 
 
+def test_normalization() -> None:
+    """Verify that the last modality can be sampled when its cumulated sum is greater than 1."""
+    df = pd.DataFrame(data=[[0.5, 0.5, 0.5]])
+    result = get_random_weighted_columns(df, np.random.default_rng(4))
+    assert result.values[0] == 2
```

Comment:
I think we should add other tests to be sure of the `get_random_weighted_columns` behaviour

Reply:
what do you have in mind ?